content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

```diff
@@ -268,99 +268,99 @@ Datadog.setUserInfo(id: "1234", name: "John Doe", email: "john@doe.com")
+`appHangThreshold`
+: Sets the threshold for reporting when an app hangs. The minimum allowed value for this option is `0.1` seconds. To disable app hangs reporting, set this to `nil`. For more information, see [Add app hang reporting][10].
 
-`uiKitActionsPredicate`
-: Enables tracking user interactions (taps) as RUM actions. You can use the default implementation of `predicate` by setting the `DefaultUIKitRUMActionsPredicate` or implement [your own `UIKitRUMActionsPredicate`](#automatically-track-user-actions) customized for your app.
+`applicationID`
+: The RUM application identifier.
 
-`urlSessionTracking`
-: Enables tracking `URLSession` tasks (network requests) as RUM resources. The `firstPartyHostsTracing` parameter defines hosts that are categorized as `first-party` resources (if RUM is enabled) and have tracing information injected (if tracing feature is enabled). The `resourceAttributesProvider` parameter defines a closure to provide custom attributes for intercepted resources that is called for each resource collected by the RUM iOS SDK. This closure is called with task information and may return custom resource attributes or `nil` if no attributes should be attached.
+`customEndpoint`
+: A custom server URL for sending RUM data.
 
-`trackFrustrations`
-: Determines whether automatic tracking of user frustrations should be enabled. By default, this is set to `true`.
+`errorEventMapper`
+: Sets the data scrubbing callback for errors. This can be used to modify or drop error events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
 
-`trackBackgroundEvents`
-: Determines whether RUM events should be tracked when no view is active. By default, this is set to `false`.
+`longTaskEventMapper`
+: Sets the data scrubbing callback for long tasks. This can be used to modify or drop long task events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
 
 `longTaskThreshold`
 : The threshold for RUM long tasks tracking (in seconds). By default, this is sent to `0.1` seconds.
 
-`appHangThreshold`
-: Sets the threshold for reporting app hangs. The minimum allowed value for this option is `0.1` seconds. To disable app hangs reporting, set this to `nil`. For more information, see [Add app hang reporting][10].
+`onSessionStart`
+: Enter a method that gets called when RUM starts the session.
 
-`trackWatchdogTerminations`
-: This setting determines whether the SDK should track application termination by Watchdog. The default setting is `false`.
+`resourceEventMapper`
+: Sets the data scrubbing callback for resources. This can be used to modify or drop resource events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
```

Comment:
```suggestion
: The data scrubbing callback for resources. This can be used to modify or drop resource events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
```